code/standards/checkstyle.xml

```diff
@@ -75,13 +83,21 @@
     <module name="FinalClass"/>
     <module name="HideUtilityClassConstructor"/>
     <module name="InterfaceIsType"/>
-    <module name="VisibilityModifier"/>
+    <module name="VisibilityModifier">
+      <property name="protectedAllowed" value="true" />
+      <property name="allowPublicFinalFields" value="true" />
```

Comment:
I'd rather have a separate check for "mutable public final" fields. If that check can't exist then it may make sense to turn this off.